network.ts

```diff
@@ -43,26 +43,25 @@ export const proxies: TablelandNetworkConfig = {
   "local-tableland": "0xe7f1725e7734ce288f8367e1bb143e90bb3f0512",
 };
 
-const homesteadURI = "https://tableland.network/api/v1/tables/1/";
```

Comment:
The chain Ids are used in the ERC721a `tokenURI` method. It simply appends the token / table Id to the baseUri. Wouldn't this change break that functionality?

Reply:
@sanderpick ah, i see. the use case i was considering this for was to be able to set the baseURI via a hardhat script that only uses evm-tableland, so it'd be nice to be able to import the baseURIs from `network.ts`. but, i think the TablelandDeployments [baseURIs](https://github.com/tablelandnetwork/evm-tableland/blob/469f11ab10682eee8ccbfd86a7e8784f176df4dd/contracts/utils/TablelandDeployments.sol#L104) can help there, and instead, the SDK `getBaseUrl` can provide the correct URL info. i'll just close this out.

and @joewagner sgtm, will tal in the future with #414

Reply:
if all you need is the host name, you could still use these URLs... just pull the host out. `/api/v1` will be the same for them all.